addons/account/report/account_aged_partner_balance.py

```diff
@@ -70,11 +70,11 @@ def _get_partner_move_lines(self, account_type, date_from, target_move, period_l
                 AND (account_account.internal_type IN %s)
                 AND (
                         l.reconciled IS FALSE
-                        OR l.id IN(
+                        OR l.id = ANY(ARRAY(
                             SELECT credit_move_id FROM account_partial_reconcile where max_date > %s
```

Comment:
Note I didn't add here to keep diff and change small but these two subselects could also benefit from having the company_ids in the where clause. It halves the time again for the query in this database for the largest company and makes nearly instant for smaller ones.